src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java

```diff
@@ -89,6 +89,7 @@ public SKOSImporter(Configuration config,
     public Vocabulary importVocabulary(final boolean rename,
                                        final URI vocabularyIri,
                                        final String mediaType,
+                                       final Consumer<Vocabulary> persist,
```

Comment:
Please, update the Javadoc as well. The meaning of the `persist` parameter is not clear from the method signature (at least not to me), so an explanation of what is expected of it would be helpful